src/misc/KeyManager.js

```diff
@@ -172,17 +172,22 @@ class KeyManager {
 
 	registerShortcuts(shortcuts: Shortcut[]) {
 		Keys.META.code = (client.browser === BrowserType.FIREFOX ? 224 : 91)
-		addAll(this._shortcuts, shortcuts)
 		for (let s of shortcuts) {
 			let id = this._createKeyIdentifier(s.key.code, s.ctrl, s.alt, s.shift, s.meta)
+			this._shortcuts.push({id: id, shortcut: s})
 			this._keyToShortcut[id] = s
 		}
 	}
 
 	unregisterShortcuts(shortcuts: Shortcut[]) {
```

Comment:
(not sure where to put this comment) Do `registerModalShortcuts`/`registerDesktopShortcuts`/`unregisterModalShortcuts` still work after this change?

Reply:
I guess they should, maybe just make sure